epregressions/runtests.py

```diff
@@ -398,10 +416,17 @@ def diff_text_files(file_a, file_b, diff_file):
             "(user input)=",
             "(input file)=",
             "(IDF Directory)=",
+            "(Current Working Directory)=",
             "(Current Working Directory)\"=",
             "ReadVars Run Time",
             "EnergyPlus Program Version",
-            "PythonPlugin: Class"
+            "PythonPlugin: Class",
+            "ExpandObjects Finished. Time:",
+            "EnergyPlus, Version",
+            "EnergyPlus Run Time=",
+            "ParametricPreprocessor Finished. Time:",
+            "ExpandObjects Finished with Error(s). Time:",
+            "Elapsed time: ",
```

Comment:
With the new standard output and standard error regressions, there are a number of strings we need to detect for ignoring when calculating the diffs.